force-app/main/default/lwc/pdftronWvInstance/pdftronWvInstance.js

```diff
@@ -44,29 +48,28 @@ export default class PdftronWvInstance extends LightningElement {
   }
 
   handleBlobSelected(record) {
-    //record = JSON.parse(record);
-
-    var blobby = new Blob([_base64ToArrayBuffer(record.body)], {
+    let blobby = new Blob([_base64ToArrayBuffer(record.body)], {
```

Comment:
```suggestion
    const blobby = new Blob([_base64ToArrayBuffer(record.body)], {
```
I don't see the variable being mutated (but I stand to be corrected)